docs/csi_capability.md

```diff
@@ -0,0 +1,48 @@
+## Controller driver capability
+
+Following matrix shows lvm-localpv controller driver capabilities.
+
+| Capability | Description | Status | Comment |
+| -------------------------------- | -------------- | ------------ | ------------ |
+| CREATE_DELETE_VOLUME | This capability indicates that the driver supports dynamic volume provisioning and deprovisioning. | Implemented |  |
+| PUBLISH_UNPUBLISH_VOLUME | This capability indicates the driver implements operations that correspond to the Kubernetes volume attach/detach operations. | Not implemented |  |
+| LIST_VOLUMES |  | Not implemented |  |
+| GET_CAPACITY | This capability indicates that the driver supports exposing available capacity of the storage pool from which the controller provisions volumes. | Implemented |  |
+| CREATE_DELETE_SNAPSHOT | This capability indicates that the driver supports provisioning volume snapshots and the ability to provision new volumes using those snapshots. | Implemented | Creation and deletion of volume snapshot is implemented but creating a volume using snapshot is not supported. |
+| LIST_SNAPSHOTS |  | Not implemented |  |
```

Comment:
For the cases where it says `Not implemented` - should we change to "Planned" and "Not Applicable" to differentiate between what will be implemented in future. 

Also in case "Not Applicable" - let us add a comment on why it is not applicable.